ingest/sqs/pools/transformer/pool_transformer.go

```diff
@@ -427,3 +460,106 @@ func (pi *poolTransformer) computeUSDCPoolLiquidityCapFromUOSMO(ctx sdk.Context,
 
 	return poolLiquidityCapUOSMO, noPoolLiquidityCapError
 }
+
+// updateAlloyTrasmuterInfo updates cosmwasmPoolModel with alloyed transmuter specific info.
+// - It queries alloyed transmuter contract asset configs and share denom, the construct
+// `AlloyTransmuterData`. Share denom for alloyed transmuter is the alloyed denom.
+// - append the alloyed denom to pool denoms.
+func (pi *poolTransformer) updateAlloyTrasmuterInfo(
```

Comment:
nit: could we move alloyed-transmute specific helpers into a separate file so that when orderbook logic is added, we don't keep bloating the same transformer file